reverse/reverse.py

```diff
@@ -0,0 +1,10 @@
+def reverse(word):
+    if word:
+        new_word = ""
+        for i, letter in enumerate(word):
```

Comment:
So while this works, perhaps consider another technique. Can you iterate through a string in reverse, take each character and append it to `new_word`?